go.mod

```diff
@@ -26,6 +26,11 @@ require (
 	gorm.io/gorm v1.21.10
 )
 
+require (
+	github.com/k0kubun/colorstring v0.0.0-20150214042306-9440f1994b88 // indirect
+	github.com/ulikunitz/xz v0.5.10
+)
```

Comment:
Can you put the necessary packages at the top and the indirectly called packages at the bottom?

Reply:
sure, I hadn't noticed.
I just used `go get` command